api/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ApiConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
This seems to also be possible to set under `address/settings.py` last line.

Is there some difference between setting here vs there?

Reply:
For a Django project having a single application setting this under 'address/settings.py' or having it in apps.py won't make any difference. 
For projects having multiple applications, if we define the 'default_auto_field' in settings.py, it will be act as a Global variable and models of all applications will have the default auto field. 
When we want models of one application have a different auto field than the other then we can set it at apps.py.